portfolio/src/main/webapp/script.js

```diff
@@ -39,33 +28,114 @@ window.onload = loadComments();
 //Retrieves json comments from server 
 function loadComments(){
     
-    var commentsLimit = commentAmount();
+    const commentsLimit = commentAmount();
+    const order = document.getElementById("order").value;
 
-    fetch('/data?number=' + commentsLimit).then(response => response.json()).then((comments) => {
+    fetch('/data?number=' + commentsLimit + '&order=' + order).then(response => response.json()).then((comments) => {
       const allCommentsList = document.getElementById('comments-container');
       allCommentsList.innerHTML = '';
       
       for(var i = 0; i < commentsLimit; i++){
-        let singleComment = document.createElement('p');
-        singleComment.innerText = comments[i].comment;
-        allCommentsList.append(singleComment);
+        allCommentsList.append(createSingleComment[i]);
       }
     });
 }
 
+function createSingleComment(comment){
+    const commentDiv = document.createElement('div');
+
+    const nameTitle = document.createElement('h3');
+    nameTitle.innerHTML = "Name: " + sanitizeHTML(comment.name);
+    commentDiv.appendChild(nameTitle);
+
+    const commentContent = document.createElement('p');
+    const moodReceived = comment.mood;
+    if (moodReceived === "happy"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😊 ";
+    } else if(moodReceived === "heart"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😍 ";
+    } else if(moodReceived === "surprised"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😯 ";
+    } else if(moodReceived === "sad"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😥 ";
+    } else {
```

Comment:
https://google.github.io/styleguide/jsguide.html#non-ascii-characters

```For the remaining non-ASCII characters, either the actual Unicode character (e.g. ∞) or the equivalent hex or Unicode escape (e.g. \u221e) is used, depending only on which makes the code easier to read and understand.
```

Surprisingly, the Google style guide agrees with you here. I would have assumed you'd have wanted:

```
const emojiChars = {
  happy : 'U+1F60A',
  ...
};
```

but this passes the guide.